js/package.json

```diff
@@ -85,8 +85,7 @@
       "@jupyter-widgets/base": {
         "bundled": false,
         "singleton": true
-      },
-      "d3": false
```

Comment:
Do we need to remove this?

Reply:
yes!

Reply:
Hahah, I thought you reminder me to remove this, I misunderstood.
Yes, we can remove this because we can share d3 now.